### PR TITLE
Added Rating,Range & Button to Product Page

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review and provide feedback

### CODE REVIEW

File: Product/Product.css [hunk 1]
```diff
@@ -170,3 +186,17 @@
     top: 1%;
     color: var(--tertiary-color);
   }
+  .primary-button-icon-cart{
+    margin-left: 2.3rem;
+    display: flex;
+    background-color: black;
+    color: white;
```
Comment (m26unkwn) [inline Product/Product.css]: You can use here variables

**Review by m26unkwn — APPROVED** [review]

File: Product/Product.css (hunk 1 above)
Comment (ShrutiLakhani) [reply Product/Product.css]: Done

File: Cart/common.css [hunk 2]
```diff
@@ -4,20 +4,16 @@ body
     box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
     font-family: Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
 }
-a > .fa 
+.icon-ecomm a > .far , .fas
 {
```
Comment (j24m) [inline Cart/common.css:8]: you can use beautify for proper indentation : )

File: Product/Product.css [hunk 3]
```diff
@@ -62,7 +62,23 @@
     margin-left:0px;
     font-size: 14px;
 }
-
+.price-input{
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
```
Comment (dipen2000) [inline Product/Product.css:68]: Someone in my PR review suggested me that if you have measurements less than 1 rem then use them but I have not looked into it yet so just passing down the suggestion to you.

**Review by j24m — APPROVED** [review]
Hello, the changes that I would suggest is to provide with proper indentation in all the files for better readability, code is good, class names are good too, and UI looks beautiful : ) 
Most importantly 5 stars for a small PR : )

File: Product/common.css [hunk 4]
```diff
@@ -4,21 +4,21 @@ body
     box-shadow: 0 4px 12px 0 rgb(0 0 0 / 5%);
     font-family: Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
 }
-a > .fa 
+:root 
+{   
+    --drop-hover-color: #ddd;
```
Comment (dipen2000) [inline Product/common.css:9]: People told me to use uppercae for the variables so just passing down that suggestion , it;s just a preference thing.

**Review by dipen2000 — APPROVED** [review]
Looks good to go!

File: Cart/common.css (hunk 2 above)
Comment (ShrutiLakhani) [reply Cart/common.css:8]: Thank You. I will make the changes

File: Product/Product.css (hunk 3 above)
Comment (ShrutiLakhani) [reply Product/Product.css:68]: Thanks

File: Product/common.css (hunk 4 above)
Comment (ShrutiLakhani) [reply Product/common.css:9]: ohh okay. I will check. But for now, keeping it like this only. Thanks for the suggestion

**Pull request merged**